hypertension/dc-7343-algorithms/py-root/lambda_app.py

```diff
@@ -0,0 +1,6 @@
+from lib.main import main  # pragma: no cover (main is tested elsewhere)
+
+
+# pylint: disable=unused-argument
+def lambda_handler(event, context):  # pragma: no cover
+    return main(event)
```

Comment:
I forgot if you mentioned if this has been tested in Lambda. I had to modify the way I used the `event` param in Lambda compared to a POSTMAN request through Flask. Specifically, accessing the params through `event['body']`. Outside of that though 👍

Reply:
Ya I should add that in as a comment.